lua/starfall/libs_sv/wire.lua

```diff
@@ -308,6 +308,43 @@ function wire_library.adjustOutputs(names, types)
 	WireLib.AdjustSpecialOutputs(ent, names, types)
 end
 
+--- Creates/Modifies wire inputs/outputs. All wire ports must begin with an uppercase
+-- letter and contain only alphabetical characters.
+-- @param inputs A key-value table with input port names as keys and types as values. Can be nil to not affect input ports.
+-- @param outputs A key-value table with output port names as keys and types as values. Can be nil to not affect output ports.
+function wire_library.adjustPorts(inputs, outputs)
+	if inputs then checkluatype(inputs, TYPE_TABLE) end
+	if outputs then checkluatype(outputs, TYPE_TABLE) end
+
+	if inputs then
```

Comment:
I'd change this to `inputs~=nil` and move the checkluatype above into this if block. Same for outputs.

Reply:
I can change it to `inputs~=nil` but the reason I did not move `checkluatype` into the if blocks is if one of the `checkluatype` checks fail the function does not continue and no code will be run otherwise half of the code would run only or none of the code at all depending on the parameters and then make people wonder what's happening there

Reply:
The Throw inside of the loop would lead to that same behavior. I think it's fine though, I can't imagine an error getting thrown and inputs being modified would lead to any confusion.

Reply:
Also, the adjustInputs and adjustOutputs function calls can throw